common/dynamic_country_names/imperia_dynamic_country_names.txt

```diff
@@ -311,4 +311,21 @@ BUL = { # Danubian Vilayet
 			}
 		}
 	}
+}
+
+SPA = {
+	dynamic_country_name = {
+		name = spa_carlist_spain
+		adjective = spa_carlist_spain_adj
+		
+		is_main_tag_only = yes
+		priority = 1
+		
+		trigger = {
+			c:SPA = {
```

Comment:
I know it's counterintuitive, but you can't assume SPA exists in this context, triggers are independent of the associated country. Should be c:SPA ?=